### PR TITLE
Docs enhancements

### PR DESCRIPTION
**Pull request by pawamoy**
Issue #521 

Changes are cleanly separated into several commits.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -25,7 +25,24 @@ nav:
...
 theme:
-  name: "material"
+  name: material
+  features:
+    - navigation.top
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: indigo
+      accent: teal
```
Comment (pawamoy) [inline mkdocs.yml:35]: I selected colors that wouldn't change too much compared to the current ones. Feel free to pick others: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette

Comment (Yajo) [reply mkdocs.yml:35]: It seems to render dark links in a hard to read color, can you fix that please? 

Screenshot_20220109-074800_Firefox

**Review by Yajo — CHANGES_REQUESTED** [review]
I see a couple of problems, and also these not related to your pr but easy to fix: 

Ugly overflow on phone: 
Screenshot_20220109-075340_Firefox

Can't see the wires when dark: 

Screenshot_20220109-075459_Firefox

These are lies now: 

Screenshot_20220109-075642_Firefox

**Comment by Yajo** [comment]
Fixed some problems myself; others can be fixed later, so merging.

**Pull request merged**